### PR TITLE
ToNi added a file with to changes

### PR DESCRIPTION
**Pull request by nikoton**
This is a pull request : meaning that some actions have to be taken before and during the 
injection of this changes into the master branch.

So this have to be understood also like a MERGE request.

### CODE REVIEW

**Comment by nikoton** [comment]
THis a comment of the pull request

File: _posts/0000-12-17-tn.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+layout:slide
+title: "Hallo Frankfurt"
```
Comment (nikoton) [inline _posts/0000-12-17-tn.md:2]: click the blue sign

Comment (nikoton) [inline _posts/0000-12-17-tn.md:2]: my review is ok ->

**Review by nikoton — COMMENTED** [review]
nothink to do

File: _posts/0000-12-17-tn.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+layout:slide
```
Comment (nikoton) [inline _posts/0000-12-17-tn.md:1]: review a sth

**Review by nikoton — COMMENTED** [review]
submit review

**Review by wt818 — APPROVED** [review]
Seems to be a good idea

**Pull request merged**